tests/acceptance/async_test_base.py

```diff
@@ -92,10 +94,12 @@ def on_open_error(self, connection, error):
 
     def on_timeout(self):
         """called when stuck waiting for connection to close"""
-        self.logger.info('on_timeout called at %s', datetime.utcnow())
-        # Initiate cleanup
-        self.timeout = None  # the dispatcher should have removed it
```

Comment:
@gst: this merge went bad. We need this line before `self.stop()`:

`self.timeout = None  # the dispatcher should have removed it`